modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectEntryLocalServiceImpl.java

```diff
@@ -1073,7 +1073,8 @@ public ObjectEntry updateObjectEntry(
 		_startWorkflowInstance(userId, objectEntry, serviceContext);
 
 		_deleteFileEntries(
-			values, objectEntry.getObjectDefinitionId(), transientValues);
```

Comment:
@marcelabc why do we need to pass the objectEntry.getValues() now?

Reply:
I fixed, thanks!